Content.Server/GameObjects/Components/Singularity/SingularityComponent.cs

```diff
@@ -169,11 +170,16 @@ public void PullUpdate()
                 controller.StopPull();
             }
             _previousPulledEntities.Clear();
+        }
 
+        public void PullUpdate()
+        {
+            CleanupPulledEntities();
             var entitiesToPull = Owner.EntityManager.GetEntitiesInRange(Owner.Transform.Coordinates, Level * 10);
             foreach (var entity in entitiesToPull)
             {
                 if (!entity.TryGetComponent<PhysicsComponent>(out var collidableComponent)) continue;
```

Comment:
God singulo's janky.

Reply:
To be honest given what I saw lurking in the RobustToolbox physics controllers abomination I think this is actually the better way, not that I'm happy with it (and what the hell happens if there's >1 singulo?)